content/millennium/r4/base/workflow/slot.md

```diff
@@ -42,6 +43,7 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/St
  ID                    | Value\[x] Type | Description
 -----------------------|----------------|------------------------------------------------------------------------------------------------
  `scheduling-location` | [`Reference`]  | Reference to the location corresponding to this schedule, where the appointment can be booked.
+ `general-resource`    | [`String`]     | Value String referencing to the General Resource corresponding to this schedule.
```

Comment:
nit: remove the 'to' and replace 'the' with 'a' so it reads like this: Value String referencing a General Resource corresponding to this schedule.